diff --git a/main.js b/main.js
index 1d6eb8e..d7ba5b8 100644
--- a/main.js
+++ b/main.js
@@ -4,13 +4,11 @@ config = {
         "backbone":                 "components/backbone/backbone",
         "backbone.browserStorage":  "components/backbone.browserStorage/backbone.browserStorage",
         "backbone.overview":        "components/backbone.overview/backbone.overview",
-        "bootstrap":                "components/bootstrap/dist/js/bootstrap",                  // XXX: Only required for https://conversejs.org website
         "converse-dependencies":    "src/deps-website",
         "converse-templates":       "src/templates",
         "eventemitter":             "components/otr/build/dep/eventemitter",
         "jquery":                   "components/jquery/dist/jquery",
         "jquery.browser":           "components/jquery.browser/dist/jquery.browser",
-        "jquery.easing":            "components/jquery-easing-original/jquery.easing.1.3", // XXX: Only required for https://conversejs.org website
         "moment":                   "components/momentjs/moment",
         "strophe":                  "components/strophe/strophe",
         "strophe.disco":            "components/strophe.disco/index",
@@ -174,9 +172,9 @@ var initializeEasing = function () {
 };
 
 if (typeof(require) !== 'undefined') {
+    define("jquery", [], function() { return jQuery; });
     require.config(config);
     require(["jquery", "converse"], function($, converse) {
         window.converse = converse;
-        initializeEasing(); // Only for https://conversejs.org website
     });
 }
diff --git a/src/build.js b/src/build.js
index 8eea814..af2d3d7 100644
--- a/src/build.js
+++ b/src/build.js
@@ -17,7 +17,6 @@
         "converse-dependencies":    "src/deps-full",
         "converse-templates":       "src/templates",
         "eventemitter":             "components/otr/build/dep/eventemitter",
-        "jquery":                   "components/jquery/dist/jquery",
         "jquery.browser":           "components/jquery.browser/dist/jquery.browser",
         "moment":                   "components/momentjs/moment",
         "strophe":                  "components/strophe/strophe",
