diff --git a/main.js b/main.js
index a5de646..9813c08 100644
--- a/main.js
+++ b/main.js
@@ -105,6 +105,7 @@ var initializeEasing = function () {
 };
 
 if (typeof(require) !== 'undefined') {
+    define("jquery", [], function() { return jQuery; });
     require.config(config);
     require(["jquery", "converse"], function($, converse) {
         window.converse = converse;
diff --git a/src/build.js b/src/build.js
index 43f2d11..377154b 100644
--- a/src/build.js
+++ b/src/build.js
@@ -11,7 +11,6 @@
         }
     },
     paths: {
-        "jquery": "components/jquery/dist/jquery",
         "jed": "components/jed/jed",
         "locales": "locale/locales",
         "af": "locale/af/LC_MESSAGES/af",
@@ -42,8 +41,8 @@
         "bigint": "src/bigint",
         "crypto.core": "components/otr/vendor/cryptojs/core",
         "crypto.enc-base64": "components/otr/vendor/cryptojs/enc-base64",
-        "crypto.md5": "components/crypto-js-evanvosberg/src/md5",
-        "crypto.evpkdf": "components/crypto-js-evanvosberg/src/evpkdf",
+        "crypto.md5": "components/crypto-js/src/md5",
+        "crypto.evpkdf": "components/crypto-js/src/evpkdf",
         "crypto.cipher-core": "components/otr/vendor/cryptojs/cipher-core",
         "crypto.aes": "components/otr/vendor/cryptojs/aes",
         "crypto.sha1": "components/otr/vendor/cryptojs/sha1",
