diff --git a/main.js b/main.js
index d374d85..bf19c57 100644
--- a/main.js
+++ b/main.js
@@ -1,3 +1,4 @@
+define("jquery", [], function() { return jQuery; });
 require.config({
     paths: {
         "jquery": "components/jquery/jquery",
diff --git a/src/build.js b/src/build.js
index c6524c4..688f171 100644
--- a/src/build.js
+++ b/src/build.js
@@ -4,7 +4,6 @@
     out: "../builds/converse.min.js",
     include: ['main'],
     paths: {
-        "jquery": "components/jquery/jquery",
         "jed": "components/jed/jed",
         "locales": "locale/locales",
         "af": "locale/af/LC_MESSAGES/af",
@@ -33,8 +32,8 @@
         "bigint": "src/bigint",
         "crypto.core": "components/otr/vendor/cryptojs/core",
         "crypto.enc-base64": "components/otr/vendor/cryptojs/enc-base64",
-        "crypto.md5": "components/crypto-js-evanvosberg/src/md5",
-        "crypto.evpkdf": "components/crypto-js-evanvosberg/src/evpkdf",
+        "crypto.md5": "components/crypto-js/src/md5",
+        "crypto.evpkdf": "components/crypto-js/src/evpkdf",
         "crypto.cipher-core": "components/otr/vendor/cryptojs/cipher-core",
         "crypto.aes": "components/otr/vendor/cryptojs/aes",
         "crypto.sha1": "components/otr/vendor/cryptojs/sha1",
